pkg/apis/application/v1alpha1/applicationset_types.go

```diff
@@ -459,11 +459,12 @@ type PullRequestGenerator struct {
 	GitLab          *PullRequestGeneratorGitLab          `json:"gitlab,omitempty" protobuf:"bytes,2,opt,name=gitlab"`
 	Gitea           *PullRequestGeneratorGitea           `json:"gitea,omitempty" protobuf:"bytes,3,opt,name=gitea"`
 	BitbucketServer *PullRequestGeneratorBitbucketServer `json:"bitbucketServer,omitempty" protobuf:"bytes,4,opt,name=bitbucketServer"`
+	AzureDevOps     *PullRequestGeneratorAzureDevOps     `json:"azuredevops,omitempty" protobuf:"bytes,5,opt,name=azuredevops"`
 	// Filters for which pull requests should be considered.
-	Filters []PullRequestGeneratorFilter `json:"filters,omitempty" protobuf:"bytes,5,rep,name=filters"`
+	Filters []PullRequestGeneratorFilter `json:"filters,omitempty" protobuf:"bytes,6,rep,name=filters"`
 	// Standard parameters.
-	RequeueAfterSeconds *int64                 `json:"requeueAfterSeconds,omitempty" protobuf:"varint,6,opt,name=requeueAfterSeconds"`
-	Template            ApplicationSetTemplate `json:"template,omitempty" protobuf:"bytes,7,opt,name=template"`
+	RequeueAfterSeconds *int64                 `json:"requeueAfterSeconds,omitempty" protobuf:"varint,7,opt,name=requeueAfterSeconds"`
+	Template            ApplicationSetTemplate `json:"template,omitempty" protobuf:"bytes,8,opt,name=template"`
```

Comment:
Since the GRPC API is technically public, this struct should be append-only. Could you reorder to add `AzureDevOps` to the end and reorder the protobuf field numbers so that none change from `master`?

(We've probably broken backwards-compatibility in the past, but we should avoid it going forward.)

Reply:
Done, thx!